### DISCUSSION TITLE
surveymarks/lx4216/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#23] [Announcements]
# surveymarks/lx4216/

http://localhost:4000/surveymarks/lx4216/

**Comment by jengalas** [comment]
Posted by Jerry
January 29, 2024 at 8:40 pm
_Imported from WordPress_

In the 1960,s the light from the High Tor beacon was visible from an upstairs window in the house I lived in at that time in Nanuet, Clarkstown, Rockland County. It was also visible for miles up and down the Hudson River from a boat. Not much left of the mountain there due to mining.
